cmd/docker/docker.go

```diff
@@ -309,6 +348,22 @@ func runDocker(dockerCli *command.DockerCli) error {
 	return cmd.Execute()
```

Comment:
```suggestion
	ctx, span := otel.Tracer("github.com/docker/docker/cli/cmd/docker").Start(
		cmd.Context(),
		"runDocker",
	)
	defer span.End()
	err = cmd.ExecuteContext(ctx)
	if err != nil {
		span.SetStatus(codes.Error, "command errored")
		span.RecordError(err)
	}
	return err
```

It'd be even better to start the span at the top of the function, so the latency before the command is executed gets captured in the traces.